week2/task2/Program.cs

```diff
@@ -0,0 +1,26 @@
+﻿// See https://aka.ms/new-console-template for more information
+Console.WriteLine("Hello, World!");
+
+var time2 = new JupiterTime(17, 88);
+PrintTime(time2);
+
+void PrintTime(JupiterTime time)
+{
+    Console.WriteLine($"({time.Hours:00}:{time.Minutes:00})");
+
```

Comment:
Try avoiding empty lines if they are not used to separate functionality or logic 😊